lib/router.js

```diff
@@ -4,38 +4,25 @@
 import version from './services/version';
 import Spatial from './services/spatial';
 import Summary from './services/summary';
+import logger from './util/logger';
 
 function js(req, res, next) {
-  req.setEncoding('binary');
-  req.on('data', (chunk) => {
-    // console.log("chunk", typeof chunk) //TODO: log this
-  });
+  // req.setEncoding('binary');
```

Comment:
Remove commented code, or replace it with a comment if this is intended as documentation.